meta/documents/user_guide/en/obtaining_credentials.md

```diff
@@ -0,0 +1,44 @@
+# Wayfair plugin: Getting API Credentials
+
+In order for the Wayfair plugin to connect to Wayfair's systems, you need to provide your unique API credentials.
+
+To receive the API credentials for you organization, follow these steps:
+
+## 1. Send an Email to Wayfair
+
+Send an email to ERPSupport@wayfair.com to request assistance:
+
+- Subject : Access to Plentymarkets plugin / "Name of your company" (SuID)
+- Body:
+    - Contact information
+    - Descriptions of your organizations' plans and needs
+    - Any important dates regarding Plentymarkets >> Wayfair integrations.
+
+You will promptly receive a response containing these details:
+- Confirmation of access to
+- Supplier ID(s)
+
+## 2. Generate Application Credentials
+
+1. Log in to your Partner Home account at partners.wayfair.com
+
+2. Find the `Developer` menu. It may be on the top banner of the page. Otherwise, it is accessible in the `More` menu in the banner.
+
+3. Activate the `Developer` menu, and click on `Application`. You should be redirected to a new page.
+
+4. On the `Application Management` page, click the `+ New Application` button at the bottom.
+
+5. Provide a `Name` and `Description` for the new Application

```

Comment:
Can we provide them an example name and description they could use?

Reply:
good idea.

Reply:
Yeah, we can provide guidance on application name (Supplier Name + Plentymarkets App).
E.g Nokia Plentymarkets App